.gitignore

```diff
@@ -4,7 +4,8 @@
 # node cache/logs
 npm-debug.log
 node_modules
+design/node_modules
 
 # statinamic builded files
-/dist
+dist
```

Comment:
Sounds good, those are minified so it makes sense that the linter would error that.